gruntfile.js

```diff
@@ -250,30 +250,19 @@ module.exports = function (grunt) {
       }
     },
 
-    autoprefixer: {
+    // Run PostCSS Autoprefixer on any CSS in the assets directory
+    // Using the configured Autoprefixer options (defaults to last 2 versions)
+    postcss: {
       options: {
-        browsers: ["last 2 versions", "ie >= 9"],
-        map: {
-          inline: false
-        }
-      },
-      patternlibrary: {
-        files: [{
-          expand: true,
-          flatten: true,
-          src: build("/pattern-library/assets/css/*.css"),
-          dest: build("/pattern-library/assets/css/")
-        }]
-      },
-      patterns: {
-        files: [
-          {
-            expand: true,
-            flatten: true,
-            src: build("/css/*.css"),
-            dest: build("/css/")
-          }
+        map: true,
+        processors: [
+          require('autoprefixer')(
```

Comment:
We should always be using double-quotes unless we are attempting to escape the quotes.